src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/arm/convolution_backprop_data.cpp

```diff
@@ -81,48 +81,4 @@ INSTANTIATE_TEST_SUITE_P(nightly_arm_Deconv_2D_Planar_FP32,
                                             ::testing::ValuesIn(filterCPUInfo({conv_gemm_2D, conv_gemm_acl_2D, conv_gemm_acl_2D_nspc})),
                                             ::testing::Values(empty_plugin_config)),
                          DeconvolutionLayerCPUTest::getTestCaseName);
-
-/* ============= Deconvolution auto padding tests ============= */
-const std::vector<DeconvInputData> inputs_2D_AutoPadding = {
-        DeconvInputData{InputShape{{}, {{2, 67, 7, 7}}}, ov::test::utils::InputLayerType::CONSTANT, {}},
-        DeconvInputData{InputShape{{-1, 67, -1, -1}, {{1, 67, 9, 4}, {2, 67, 5, 7}, {1, 67, 9, 4}}},
-                        ov::test::utils::InputLayerType::CONSTANT,
-                        {}},
-        DeconvInputData{InputShape{{-1, 67, -1, -1}, {{2, 67, 7, 7}, {2, 67, 5, 7}, {1, 67, 9, 4}}},
-                        ov::test::utils::InputLayerType::CONSTANT,
-                        {}},
-        DeconvInputData{InputShape{{-1, 67, -1, -1}, {{1, 67, 9, 4}, {2, 67, 5, 7}, {1, 67, 9, 4}}},
-                        ov::test::utils::InputLayerType::PARAMETER,
-                        {}}};
-
-const auto deconvParams_AutoPadding_2D =
```

Comment:
The test name is AutoPadding and it is also testing the inputs with auto padding. Are we sure we are not removing an extra test scope together with blocking layout?

Reply:
2D AutoPadding tests are already presented in `x64` directory:
- nightly_Deconv_2D_AutoPadding_FP32
- smoke_Deconv_2D_AutoPadding_AMX_BF16
- smoke_Deconv_2D_AutoPadding_AMX_FP16

Reply:
Here we are changing scope for ARM, not x84
What about AutoPadding tests for ARM architecture?

Reply:
There were 2D AutoPadding tests with blocked layout only (before tests refactoring). So, the scope of deconv test coverage is not reduced by this change.